planner/core/memtable_predicate_extractor.go

```diff
@@ -472,6 +472,20 @@ func (helper extractHelper) parseQuantiles(quantileSet set.StringSet) []float64
 	return quantiles
 }
 
+func (helper extractHelper) parseQuantilesUint64(quantileSet set.StringSet) []uint64 {
+	quantiles := make([]uint64, 0, len(quantileSet))
+	for k := range quantileSet {
+		v, err := strconv.ParseUint(k, 10, 64)
+		if err != nil {
+			// ignore the parse error won't affect result.
```

Comment:
Do we need a log here?

Reply:
add `log.Info("faild to parse uint64 from string", zap.String("string:", k))` for `parseQuantilesUint64`.
add add `log.Info("faild to parse float64 from string", zap.String("string:", k))` for `parseQuantiles`.